readme.md

```diff
@@ -390,6 +390,7 @@
 - [Podcasts](https://github.com/guipdutra/awesome-geek-podcasts#readme)
 - [Email Newsletters](https://github.com/vredniy/awesome-newsletters#readme)
 - [IT Quotes](https://github.com/victorlaerte/awesome-it-quotes#readme)
+- [IT-Films](https://github.com/greybax/awesome-it-films#readme)
```

Comment:
> The entry in the Awesome list should:
> * Include a short description about the project/theme of the list. **It should not describe the list itself.**

Reply:
Not sure what do you mean?

Description is 
>A curated list of awesome movies, documentaries and TV series about hackers, geeks and IT in common, sorted by IMDB rating. If you know some good films which do not contain in these lists just fork it and send a pull request.

And its already has on https://github.com/greybax/awesome-it-films#readme page.

Reply:
The description here could then be something like this:

> Movies, documentaries and TV series about hackers, geeks and information technology.

Reply:
added description (see changes https://github.com/sindresorhus/awesome/pull/1434/commits/3deac218a8d84a3d25154b9144098ef64f2484d7)